### PR TITLE
Update README 

### PR DESCRIPTION
**Pull request by tonykipkemboi**
1.  Updated all instances of `crewAI` to `CrewAI`
2. Updated the `Setting Your Crew` example with a full working example following the YAML config format. This one specifically should improve DevX and onboarding with a real working example end-to-end.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,83 +44,223 @@ To get started with CrewAI, follow these simple steps:
...
 
 ```shell
 pip install 'crewai[tools]'
 ```
+The command above installs the basic package and also adds extra components which require more dependencies to function.
+
+Lock the dependencies and install them by using the CLI command:
+
+```shell
+crewai install
```
Comment (theCyberTech) [inline README.md]: I think this needs to be below crewai create command??

Comment (tonykipkemboi) [reply README.md]: Good point. Totally missed that.

Comment (tonykipkemboi) [reply README.md]: Updated!

File: README.md [hunk 2]
```diff
@@ -44,83 +44,224 @@ To get started with CrewAI, follow these simple steps:
...
+  role: >
+    {topic} Senior Data Researcher
+  goal: >
+    Uncover cutting-edge developments in {topic}
+  backstory: >
+    You're a seasoned researcher with a knack for uncovering the latest
+    developments in {topic}. Known for your ability to find the most relevant
+    information and present it in a clear and concise manner.
+  tools:
+    - SerperDevTool # This is a tool that allows the agent to search the web for information
```
Comment (bhancockio) [inline README.md]: We are declaring `SerperDevTool` here and in the crew.py.

I would probably drop this one and then only leave the SerperDevTool in the crew.py

Comment (tonykipkemboi) [reply README.md]: Updated.

**Pull request merged**